SalesforceFoundation/EDA
# Account Look up Auto-Population

# Critical Changes

# Changes

## General Updates
We've fixed an issue that occurs when the Account lookup on a Contact is cleared. With this fix, the lookup is repopulated with the Account (if the Account is the default record type), or is populated with a new Account record of the default record type.

# Issues Closed
Fixes #557 

# New Metadata

# Deleted Metadata

# Testing Notes

- Create a contact. The Account lookup is auto-populated with the Account with default account model record type.
- Clear the Account lookup field. The system should re-populate the Account lookup with the same Account.

- Create a contact and manually populate the Account lookup to an Account that is *not* the default account model record type.
- Clear the Account lookup field. The system should create a new Account of the default account model record type and populate the Account lookup with the new Account.
